blog/posts/2024-11-07-how-to-get-a-rust-pkg-to-cran/index.qmd

```diff
@@ -0,0 +1,207 @@
+---
+title: "How to get a Rust-based Package to CRAN"
+description: |
+  This blog entry outlines the journey to get a Rust-based package to CRAN.
+author: David Zimmermann-Kollenda
+date: "11/07/2024"
+image: images/extendr-release-070.png
+image-alt: "The extendr logo, letter R in middle of gear."
+categories: [CRAN, Package, Best-Practices, rtiktoken]
+---
+
+I finally did it: I published a Rust-based package on CRAN.
+There where a couple of gotchas that I ran into, which I wanted to document here, so that your journey might be a bit faster and easier.
+
+Before I highlight what I learned, allow me to ~self-promote the backage~ talk a bit about the package first.
```

Comment:
I would love if it you started out by showing off how cool rtiktoken is and its uses. In the beginning you can say something like 

- introducing new R package rtiktoken
- this post will introduce you to the package and how tokenizers can be used
- lastly i'll talk about the journey to publishing the R package to CRAN

```suggestion
```